src/helpers/GiveRedemptionBalanceHelper.ts

```diff
@@ -18,33 +17,58 @@ interface IDonorAddresses {
 // Gets a recipient's info. Separating it out into a helper allows us to call it on addresses
 // other than the current user's without overwriting the data in Redux store. This is needed
 // to pull data on our partner projects
+/**
+ * Gets a recipient's donation info including redeemable value (denominated in sOHM),
+ * current sOHM principal directed towards them earning yield, and the gOHM equivalent
+ * value of the principal directed towards them
+ * @param address The address of the user or recipient we wish to pull this info for
+ * @param networkID ID number of network the user is currently connected to
+ * @param provider Ethereum network provider object
+ * @returns Redeemable sOHM plus recipientInfo object (totalDebt and agnosticDebt)
+ */
 export const getRedemptionBalancesAsync = async ({ address, networkID, provider }: IBaseAddressAsyncThunk) => {
   let redeemableBalance = 0;
   const recipientInfo: IUserRecipientInfo = {
-    totalDebt: "",
-    carry: "",
-    agnosticDebt: "",
-    indexAtLastChange: "",
+    totalDebt: "0",
+    agnosticDebt: "0",
   };
 
-  if (addresses[networkID] && addresses[networkID].GIVING_ADDRESS) {
+  if (!(addresses[networkID] && addresses[networkID].GIVING_ADDRESS)) {
```

Comment:
I suggest exiting the function here.

Reply:
it does doesn't it? If that condition is hit, console.log's the error message and then jumps straight to the return.

Reply:
Cleaner to just exit then and there.